includes/admin.form.inc

```diff
@@ -44,6 +44,13 @@ function islandora_usage_stats_admin_form() {
     '#default_value' => variable_get('islandora_usage_stats_exclude_bots', '1'),
   );
 
+  $form['usage']['islandora_usage_stats_exclude_bots_regex'] = array(
+    '#type' => 'textfield',
```

Comment:
Just a suggestion, but since the regex could get quite long, maybe we should add ` '#maxlength' => 256` since the default max length of textfields is 128 characters.

Reply:
Good call :+1:

Reply:
:+1: